13-students-doing-data-science.Rmd

```diff
@@ -1 +1,146 @@
+
+### Problem-based learning 
+
+Problem-based learning (PBL) is a method of instruction that presents learners with a real-world challenge in which they must apply their skills and knowledge to solve. We applied this principle to the design of this book through including walk-throughs for common questions with respect to data science and education. Whereas some topics may benefit less from such an approach (and the inclusion of walk-throughs), for data science, we believe this is important because we do not have all of the right answers in this text. Moreover, there is not one right statistical model or algorithm, a technique to write code or even software and tools to utilize. 
+
+Thus, the text features walkthroughs that reflect the types of challenges that educational data scientists may encounter in the course of their work: readers may choose to go about approaching the analysis of the data used in each walkthrough differently. At the end of walkthrough, there exist exercises that provide the opportunity for readers to extend and apply the ideas presented in the chapter. Moreover, the challenges are structured in such a way that readers return to some of them, but with different aims, over the course of the book.
+
+<!-- To what extent do we need to change the walkthroughs or their order to make this accurate? -->
+
+<!-- Wonder if we add another paragraph about how others can apply problem-based learning in any setting? -->
+
+### Differentiation
+
+Differentiation is a method for providing multiple pathways for learners to engage with, understand, and ultimately apply new content and new skills. To differentiate this text, we first created personas for who we expected to be common groups of readers of the book (see [Wilson, 2019](http://teachtogether.tech), for an example). We then differentiate the book by recognizing and providing background content/skills (either explicitly or through reference to other resources), embedded checks for understanding, and recommendations for where to begin based upon prior expertise. We also provide screenshots that are annotated and reflective of the content in the text to help to show readers how they are able to use what they are reading about.
+
+<!-- Say more about this - specifically how we use the user personas? -->
+
+Part of differentiating this book concerns for whom we are differentiating it. We consider inclusivity (in terms of who belongs as a part of the audience for this text and how this broader view of who participates in data science implies the types of challenges, topics, and data that we include in the book) and accessibility (technically, in terms of how a wide audience of readers is able to access and use the book, as well as in terms of the ways in which the content is written based on the unique assets that those in education bring) along with how we differentiate the book. 
+
+<!-- Same as above - do we say more about how other teachers can differentiate their instruction? -->
```

Comment:
Out of scope - although I think we could say more about who we think will benefit from this book (as much as we would love to write a book for _everyone_, there is a more defined persona in our audience(s).)

Reply:
know and am happy that we've started working on this!